plugins/by-name/neotest/adapters.nix

```diff
@@ -46,7 +46,7 @@ let
 
           warnings = optional (!config.plugins.treesitter.enable) ''
             Nixvim (plugins.neotest.adapters.${name}): This adapter requires `treesitter` to be enabled.
-            You might want to set `plugins.treesitter.enable = true` and ensure that the `${props.treesitter-parser}` parser is enabled.
+            You might want to set `plugins.treesitter.enable = true` and ensure that the `${treesitter-parser}` parser is enabled.
```

Comment:
Wasn't this part of the other backport PR?

Reply:
oops. sorry

Reply:
Honestly strange it was even possible, should really have been skipped automatically by git.

Reply:
I was resetting author on the commits for this PR